tests/readrows_test.go

```diff
@@ -241,7 +241,15 @@ func TestReadRows_Retry_PausedScan(t *testing.T) {
 	origReq := <-recorder
 	retryReq := <-recorder
 	if diff := cmp.Diff(clientReq, origReq.req, protocmp.Transform(), protocmp.IgnoreEmptyMessages()); diff != "" {
-		t.Errorf("diff found (-want +got):\n%s", diff)
+		origRows := origReq.req.GetRows()
+		// Check if rows or row ranges are present in requests
+		if origRows == nil || origRows.GetRowRanges() == nil || len(origRows.GetRowRanges()) == 1 {
```

Comment:
I think we should verify that getRowRanges is an empty range

Reply:
Can you clarify what you mean by that with an example?

Reply:
if we're checking len(originRows.GetRowRanges) = 1, does that mean [{keyA, keyB}] will also be valid? But we want to verify it's [{}].

Reply:
Added extra check.